fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

```diff
@@ -0,0 +1,136 @@
+---
+id: applications-cisco-ssms-restapi
+title: Cisco SSMS
+---
+
+## Vue d'ensemble
+
+Cisco Smart Software Manager On-Prem (anciennement connu sous le nom de Cisco Smart Software Manager satellite) est un composant de Cisco Smart Licensing
+qui fonctionne en conjonction avec Cisco Smart Software Manager (SSM). 
+Il offre une visibilité et des rapports en temps quasi réel sur les licences Cisco que vous achetez et consommez,
+tout en donnant aux organisations sensibles à la sécurité un moyen d'accéder à un sous-ensemble de fonctionnalités de Cisco SSM
+sans utiliser de connexion Internet directe pour gérer leur base d'installation.
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+* Applications
+* Serveurs
+* Licences
+* Alertes
+
+### Métriques collectées
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Alerts-->
+
+| Metric Name                | Description              |
+| :------------------------- | :------------------------|
+| account.alerts.minor.count | Counter of alerts minor. |
+| account.alerts.major.count | Counter of alerts major. |
+
+
+<!--Licenses-->
+
+| Metric Name                | Description                                        |
+| :------------------------- | :------------------------------------------------- |
+| licenses.usage.count       | Counter of licenses usage. Unit : Bytes (B)        |
+| licenses.free.count        | Counter of licenses free. Unit : Bytes (B)         |
+| licenses.usage.percentage  | Percentage of licenses usage. Unit : percentage(%) |
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+### Privilèges API
+
+Un compte de service est requis pour interroger l'API Cisco SSMS. Celui-ci doit avoir suffisamment de privilèges en lecture du compte ciblées.

```

Comment:
Maybe add a link to the official documentation.

Reply:
Link added.